src/types/Category.tsx

```diff
@@ -20,3 +20,17 @@ export enum TransactionType {
   income = 'Income',
   expense = 'Expense',
 }
+
+export interface Transaction {
```

Comment:
This branch should not contain Transaction model :)

Reply:
I just canceled it.